cli/cmd/install.go

```diff
@@ -399,18 +399,23 @@ func renderCRDs(ctx context.Context, k *k8s.KubernetesAPI, w io.Writer, options
 		return err
 	}
 
+	finalValues := charts.MergeMaps(defaultValues, valuesOverrides)
+
 	// If any of the Gateway API CRDs are installed, we default to rendering the
 	// Gateway API CRDs.
 	if k != nil {
 		installed, err := checkGatewayAPICRDs(ctx, k)
 		if err != nil {
 			return err
 		}
-		if installed == Absent {
+		if installed == Absent && finalValues["installGatewayAPI"] == false {
+			// if we are not installing GW API Resources and they are not present, error
 			return errors.New("The Gateway API CRDs must be installed prior to installing Linkerd: https://gateway-api.sigs.k8s.io/guides/#installing-gateway-api")
 		} else if installed == Linkerd {
+			// if they are installed and managed by Linkerd, set to true
 			defaultValues["installGatewayAPI"] = true
```

Comment:
should we also check that this has not been overriden to false  and error if this is the case?